src/Symfony/Component/Notifier/Bridge/Lox24/webhook/Lox24RequestParser.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Component\Notifier\Bridge\Lox24\Webhook;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestMatcher\MethodRequestMatcher;
+use Symfony\Component\HttpFoundation\RequestMatcherInterface;
+use Symfony\Component\RemoteEvent\Event\Sms\SmsEvent;
+use Symfony\Component\Webhook\Client\AbstractRequestParser;
+use Symfony\Component\Webhook\Exception\RejectWebhookException;
+
+/**
+ * @author Andrei Lebedev <andrew.lebedev@gmail.com>
+ */
+final class Lox24RequestParser extends AbstractRequestParser
+{
+
+    protected function getRequestMatcher(): RequestMatcherInterface
+    {
+        return new MethodRequestMatcher('POST');
+    }
+
+    /**
+     * @throws RejectWebhookException
+     */
+    protected function doParse(Request $request, #[\SensitiveParameter] string $secret): ?SmsEvent
+    {
+        // Notifications: https://doc.lox24.eu/#section/Introduction/Notifications
```

Comment:
lets move this comment to the class level

Reply:
done